asserts/asserts.go

```diff
@@ -193,17 +193,6 @@ func Decode(serializedAssertion []byte) (Assertion, error) {
 	return buildAssertion(headers, body, content, signature)
 }
 
-func checkMandatory(headers map[string]string, name string) (string, error) {
-	value, ok := headers[name]
-	if !ok {
-		return "", fmt.Errorf("assertion %v header is mandatory", name)
-	}
-	if len(value) == 0 {
-		return "", fmt.Errorf("assertion %v should not be empty", name)
-	}
-	return value, nil
-}
-
 func checkInteger(headers map[string]string, name string) (int, error) {
```

Comment:
Cool, I'd still suggest moving both of them there, as it'll be the most expected place to look for them, makes it easy to see all checkers at once, and as a bonus it takes less relevant logic away from this file. Might be good to have a `default int` parameters in checkIntegers indeed; will make even the current use case more clear.